.github/workflows/publish_release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
 
       - name: Install dependencies
-        run: pip install -r requirements.txt
+        run: pip install -r requirements.txt -r requirements-test.txt
```

Comment:
So this works, but we also have a local action for [installing dependencies](https://github.com/wayfair-incubator/columbo/blob/6c05a95816e3eb3ec1fc81e3f51b40bc363c845a/.github/workflows/main.yml#L70). I think it wasn't used here initially was because I first implemented it in a personal repo and I forgot to update it when coping over.

Reply:
Perfect, thanks!